lib/puppet_x/vmware/vmware_lib/puppet_x/puppetlabs/transport.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Wrapper to load the transport library from vmware_lib
```

Comment:
I think this is a good comment to clarify what these files are attempting to do. It would be nice if we could get a similar comment in the other new additions in lib/puppet_x/vmware/vmware_lib